src/services/expenses.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const generateId = () => {
+  if (expenses.length < 1) {
+    return 1;
+  }
+
+  const maxId = Math.max(...expenses.map(expense => expense.id));
```

Comment:
This will lead to issues. E.g you delete id=5, then create a new entity, and it will be assigned to an id that was deleted.